test/puppetlabs/puppetdb/cli/services_test.clj

```diff
@@ -139,6 +139,15 @@
                    :certname "foo.local"}}
                 (set result))))))))
 
+(deftest query-with-explain
+  (svc-utils/with-single-quiet-pdb-instance
+      (check-service-query
+       :v4 ["from" "facts" ["=" "certname" "foo.local"]]
+       {:explain? true}
```

Comment:
The error from the comment before with `{:explian? false}` it's only a testing issue and I don't think we need to update the test helper. The `check-service-query` func calls `(realized? result-set)` and the `result-set` is an empty vec when nothing is returned instead of the lazy-seq that's returned where there are results.